src/modules/Classroom/containers/ClassroomView.tsx

```diff
@@ -1,14 +1,125 @@
+import { Accordion, Button } from 'flowbite-react';
```

Comment:
can you move this into a `Classroom/containers/Classroom/index.ts` or `Classroom/containers/ClassroomOverview/index.ts` to keep the structure consistent with the course module?